hack/hooks/pre-commit.sh

```diff
@@ -39,8 +39,20 @@ if [ -n "$yaml_unformatted" ]; then
   yamlfmt $yaml_unformatted
 fi
 
+#### I18N missing statements ####
+cd frontend
+
+yarn i18n
+
+i18n_files=$(git diff --name-only HEAD --diff-filter=M | grep -E 'translation.json')
+
 #### Git commit check ####
 if [ -n "$yaml_unformatted" ] || [ -n "$go_unformatted" ]; then
   echo "Some files have been formatted - the git commit is aborted."
   exit 1
 fi
+
+if [ -n "$i18n_files" ]; then
+  echo "New i18n statements have not been committed - the git commit is aborted."
```

Comment:
Could we add instructions about how to fix it? Now I know, but maybe someone doesn't have an idea about how to fix it. We could add (Ex): "Run 'make build-ui' and include the generated translation files" (Or a link with the instructions)?

Reply:
With the verification the new i18n statements are added to the translation files, but not committed.

Is this message clearer for you?:

```
echo "New i18n statements are generated but not committed - the git commit is aborted. Please include the updated translation files in the commit"
```

Reply:
Ah, i didn't realized it already was generated! So it looks good.